toolkit/resources/manifests/package/ccache-configuration.json

```diff
@@ -32,7 +32,7 @@
       "name": "kernel",
       "comment": "",
       "enabled": true,
-      "packageNames": [ "kernel", "kernel-azure", "kernel-hci", "kernel-mshv", "kernel-uvm" ]
+      "packageNames": [ "kernel", "kernel-azure", "kernel-hci", "kernel-mshv", "kernel-uvm", "kernel-mos" ]
```

Comment:
This change seems weird because `3.0-dev` doesn't have a `kernel-mos` package. Does ccache have some sort of crossover between `main` and other branches?

Reply:
Oops, my mistake then. I believe this is needed on main, though. I'll retarget

Reply:
Will revert here, opened #6807